WowPacketParser/Store/Objects/QuestTemplate.cs

```diff
@@ -259,6 +259,9 @@ public sealed class QuestTemplate : IDataModel
         [DBFieldName("Expansion", TargetedDatabase.Legion)]
         public int? Expansion;
 
+        [DBFieldName("ManagedWorldStateID", TargetedDatabase.BattleForAzeroth)]
```

Comment:
@funjoker you gotta parse the enUS quest_template sniff again for this field

Reply:
Reeeee

Reply:
@funjoker just parsed enUS brute + all of my 815 sniffs, the field is 0 everywhere, so it's rarely used or even unused atm.